cmd/noms/diff/diff_test.go

```diff
@@ -70,7 +72,32 @@ func createStruct(name string, kv ...interface{}) types.Struct {
 	return types.NewStruct(name, fields)
 }
 
-func TestNomsMapdiff(t *testing.T) {
+func pathsFromDiff(v1, v2 types.Value, leftRight bool) []string {
+	dChan := make(chan Difference, 64)
```

Comment:
> buffer slow

Oh, and the other big disadvantage of buffered channels is that they can hide deadlock bugs.